utils/files.ts

```
@@ -119,35 +119,171 @@ export function streamToCSV (
+      currency: this.currency,
+      address: DEFAULT_MULTI_VALUES_LINE_LABEL,
+      newtworkId: tempTxGroup[0].address.networkId,
+      notes
+    } as TransactionFileData)
+  }
+
+  private extractButtonNamesFromGroupKey(groupKey: string): string {
+    return groupKey.split('_').slice(2).join(';')
+  }
+
+  private getUniquePrices(tempTxGroup: TransactionsWithPaybuttonsAndPrices[], groupKey: string): Set<number> {
```

ESLint

[error] Missing space before function parentheses.

(@typescript-eslint/space-before-function-paren)